### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**
Hi please review.

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Good folder structure
Good to go..

File: src/Context/AuthContext.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 const AuthProvider = ({children}) => {
-    const localStorageToken = JSON.parse(localStorage.getItem("token"));
-    const [token, setToken] = useState(localStorageToken?.token);
+    const localStorageToken = JSON.parse(localStorage.getItem("signup"));
+    const [token, setToken] = useState(localStorageToken);
     const localStorageUser = JSON.parse(localStorage.getItem("user"));
-    const [user, setUser] = useState(localStorageUser?.user);
+    const [user, setUser] = useState(localStorageUser);
+    const [isLogin, setIsLogin] = useState(JSON.parse(localStorage.getItem("login")) ? true : false)
```
Comment (rohanmathur91) [inline src/Context/AuthContext.js:12]: don't directly read data from local storage instead use `useEffect`, inside that read from localstorage and set the state

File: src/components/Price.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const Price = () => {
+    const { totalPrice, discountPrice } = useProducts();
+    return (
+        <div>
+            <div class="cart-sidebar-coupons flex-column">
+                <div class="cart-coupon-heading gray-text mb-2">COUPONS</div>
+                <div class="cart-coupon-container flex-row">
+                    <div class="coupon-desc flex-row gap">
+                        <div class="coupon-icon align-self-center"><i class="fas fa-tag"></i></div>
+                        <div class="align-self-center coupon-text">Apply Coupons</div>
```
Comment (rohanmathur91) [inline src/components/Price.jsx:13]: suggestion: try to use semantic HTML

File: src/index.js [hunk 3]
```diff
@@ -15,7 +14,9 @@ ReactDOM.render(
     <BrowserRouter>
       <AuthProvider>
       <ProductContext>
-        <App />
+
+          <App />
+
```
Comment (rohanmathur91) [inline src/index.js:19]: avoid spacing

**Review by rohanmathur91 — APPROVED** [review]
- Code looks clean, but try where possible to use semantic tags
- File structure is also good
- Design looks consistent throughout the pages

**Pull request merged**